server/testutils/factories/booking.ts

```diff
@@ -1,13 +1,13 @@
-import { Factory } from 'fishery'
 import { faker } from '@faker-js/faker/locale/en_GB'
 import { addDays } from 'date-fns'
+import { Factory } from 'fishery'
 
 import type { Booking } from '@approved-premises/api'
+import { DateFormats } from '../../utils/dateUtils'
 import arrivalFactory from './arrival'
-import departureFactory from './departure'
 import confirmationFactory from './confirmation'
+import departureFactory from './departure'
 import personFactory from './person'
-import { DateFormats } from '../../utils/dateUtils'
```

Comment:
As mentioned in my commit, my editor is doing some automatic linting here. Happy to remove if it's not the way? looks Looks like it's sorting it alphabetically and by case.

Reply:
I like this order of import but not sure what Bryan thinks

Reply:
I shall merge this to see if I can deploy and beg forgiveness from you @bryangaledxw, let me know if I need to turn this off.